data/Tsakalos_2022/metadata.yml

```diff
@@ -0,0 +1,328 @@
+    constrained (wet depressions) and is typical of clay-enriched or clayey soils
+    with higher exchangeable cation content. The sandy shrubland is species-rich and
+    includes diagnostic species such as Alexgeorgea nitens (Restionaceae), Banksia
+    attenuata (Proteaceae), Callitris acuminata (Cupressaceae) and Verticordia grandis
+    (Myrtaceae). The clayey shrubland has a lower species richness and includes diagnostic
+    species such as Melaleuca concreta, M. urceolaris, Thryptomene mucronulata (all
+    Myrtaceae) and Conostylis prolifera (Haemodoraceae). Classification of a species-trait
+    data set featuring response traits linked to water relationships, carbon balance,
+    nutrition acquisition and fire response revealed two functional communities also
+    distributed along with the previously identified sand-clay soil texture and coupled
+    exchangeable cation gradient (Tsakalos et al., 2019). ^p Data collection and collation
+    ^p We use species compositional, soil and trait data sets featured in Tsakalos
+    et al. (2018, 2019, 2020). The plot data collation details are given in Tsakalos
+    et al. (2018). The availability of soil data sampled and analysed at the plot
+    level (n = 189; each of 100 m2 in size) served as the limiting factor in selecting
+    and constructing balanced compositional and trait data sets. A preliminary inspection
+    of the soil and compositional properties led to the removal of 51 marginal plots.
+    These were considered disturbed - proximal to paddocks and containing edaphic
+    variables deviating from those expected in natural systems (such as P > 50 mg
+    kg-1, Singh and Gilkes, 1991; bulk density < 1.1,Bache et al., 2008) and containing
+    species associated with younger dune systems linked to exposed limestone. We used
+    138 plots positioned in the sandy (119) and clayey (19) shrubland communities
+    for all further analyses. We used soil data on bulk density (g cm-3; proxy for
+    water availability, with an inverse relationship), total N (%; N) and total P
+    (mg kg-1; P), which captured a spatially representative subset of the survey area
+    to form our environmental data set (for soil sampling and chemical extraction
+    details, refer to Tsakalos et al., 2018). ^p We created a CGO database by scoring
+    six (one main organ type per species) nominal CGO trait categories (i.e. lignotubers,
+    herbaceous and woody rhizomes, stolons, tubers, stem fragments) and recognizing
+    the absence of clonality (i.e. nonclonal) as an additional category, for all 463
+    species present within the 138 plots (Supplementary Data Table S1).
+  original_file: mcac131_suppl_supplementary_material.doc in raw data folder
+  notes: none
+locations:
+  Eneabba:
+    latitude (deg): -29.8167
+    longitude (deg): 115.2667
+    soil type: sandy plains (~85 % of the study area), laterite-capped hillocks (~10
+      %) and minor drainage lines (~5 %)
```

Comment:
Is it worth adding the sentences about vegetation type?

Reply:
done